.github/20-feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: Use this template for raising a feature request
+
+---
+
+<em>tag:feature_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version (you are using) :
+- Are you willing to contribute it (Yes/No) :
+
+
+
+**Describe the feature and the current behavior/state.**
+
+**Will this change the existing plugins? If so, please provide plugin name and details**.
+
+**Is this about adding a new plugin?**
+
+**Who will benefit with this feature?**
+
+**Is there any existing workaround for this ?**
```

Comment:
Sorry for nit: can you remove the whitespace before the question mark?

Reply:
Done